File: src/helpers/tx.js
const WebSocket = require('ws');
const httpProxy = require('http-proxy');
const tx = () => {
    const url = 'wss://api1.apiwyn88.net/gameconnector'; // Địa chỉ WebSocket
    const headers = {
        'Accept-Encoding': 'gzip, deflate, br',
        'Accept-Language': 'en-US,en;q=0.9',
        'Cache-Control': 'no-cache',
        'Connection': 'Upgrade',
        'Host': 'api1.apiwyn88.net',
        'Origin': 'https://play.wyn88.fun',
        'Pragma': 'no-cache',
        'Sec-WebSocket-Extensions': 'permessage-deflate; client_max_window_bits',
        'Sec-WebSocket-Key': '8ADzCPfoyOkX0iZJtJXBMw==',
        'Sec-WebSocket-Version': '13',
        'Upgrade': 'websocket',
        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.5672.93 Safari/537.36'
    };

    const ws = new WebSocket(url, { headers });

    ws.on('open', () => {
        console.log('WebSocket connected');
    });

    ws.on('message', (data) => {
        console.log(`Received data: ${data}`);
    });

    ws.on('close', (code, reason) => {
        console.log(`WebSocket disconnected with code: ${code} and reason: ${reason}`);
    });

    ws.on('error', (error) => {
        console.error(`WebSocket error: ${error}`);
    });
}
export default tx;

File: tx.py
import websocket

websocket.enableTrace(True)

url = 'wss://api2.apiwyn88.net/gameconnector?9a8d7aadb739258e5dcbbb69e781dee1'

headers = {
    'Host': 'api2.apiwyn88.net',
    'Connection': 'Upgrade',
    'Pragma': 'no-cache',
    'Cache-Control': 'no-cache',
    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36',
    'Upgrade': 'websocket',
    'Origin': 'https://play.wyn88.fun',
    'Sec-WebSocket-Version': '13',
    'Accept-Encoding': 'gzip, deflate, br',
    'Accept-Language': 'vi,en;q=0.9',
    'Sec-WebSocket-Key': 'GUbmkvym2NfTa5wvnP5/Kg==',
    'Sec-WebSocket-Extensions': 'permessage-deflate; client_max_window_bits'
}

def on_open(ws):
    print('Connection established')

def on_message(ws, message):
    print(f'Received message: {message}')

def on_close(ws,a,b):
    print('Connection closed',a)

ws = websocket.WebSocketApp(url, header=headers, on_open=on_open, on_message=on_message, on_close=on_close)
ws.run_forever()